service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -1765,4 +1765,31 @@ public List<LocationSummaryDto> getLocationSummary() {
             .map(location -> modelMapper.map(location, LocationSummaryDto.class))
             .collect(toList());
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    @Transactional
+    public OrderWithAddressesResponseDto makeAddressActual(Long addressId, String uuid) {
+        Address currentAddress = addressRepo.findById(addressId).orElseThrow(
+            () -> new NotFoundException(NOT_FOUND_ADDRESS_ID_FOR_CURRENT_USER + addressId));
+
+        User user = userRepository.findByUuid(uuid);
+
+        if (!currentAddress.getUser().equals(user)) {
+            throw new AccessDeniedException(CANNOT_DELETE_ADDRESS);
+        }
+
+        if (AddressStatus.DELETED.equals(currentAddress.getAddressStatus())) {
```

Comment:
for enums, we can use `==`
`AddressStatus.DELETED == currentAddress.getAddressStatus()`